content/source/docs/cloud/users-teams-organizations/single-sign-on/custom-saml.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: "cloud"
+page_title: "SAML - Single Sign-on - Terraform Cloud and Terraform Enterprise"
+---
+
+-> **Note:** Single sign-on is a paid feature, available as part of the **Business** upgrade package. [Learn more about Terraform Cloud pricing here](https://www.hashicorp.com/products/terraform/pricing/).
+
+# Single Sign-on: SAML
+
+The SAML SSO integration currently supports the following features of SAML 2.0:
+
+- Service Provider (SP)-initiated SSO
+- Identity Provider (IdP)-initiaited SSO
+- Just-in-Time Provisioning
+
+The SAML SSO integration can be configured by providing a metadata URL or manually with the Single Sign-on URL (ACS URL), Entity ID or Issuer URL, and X.509 Certificate.
+
+## Configuration (Terraform Cloud)
+
+1. Visit your organization settings page and click "SSO".
+
+2. Click "Setup SSO".
+
+   ![sso-setup](../images/sso/setup.png)
+
+3. Select "SAML" and click "Next".
+
+   ![sso-wizard-choose-provider-saml](../images/sso/wizard-choose-provider-saml.png)
+
+4. Proceed with configuring using the metadata URL or manually providing the IdP configuration.
+
+   **Metadata URL**
+
+   1. Provide your metadata URL and click the "Save settings" button.
+
+      ![sso-wizard-configure-settings-metadata](../images/sso/wizard-configure-settings-metadata.png)
+
+   **Manually**
+
+   1. Toggle the identity provider configuration.
+
+      ![sso-wizard-configure-settings-toggle](../images/sso/wizard-configure-settings-toggle.png)
+
+   2. Provide your Single Sign-on URL, Entity ID or Issuer URL, and X.509 Certificate.
```

Comment:
Updated image. 

<img width="1025" alt="Screen Shot 2020-08-25 at 2 20 05 PM" src="https://user-images.githubusercontent.com/63596285/91229110-34221b00-e6de-11ea-832e-b08d2bbc6862.png">

Reply:
Going to generate another as there seems to be focus on the toggle (box, dotted lines).